src/Compilers/CSharp/Test/Symbol/Symbols/Source/CustomModifierCopyTests.cs

```diff
@@ -269,6 +269,71 @@ class Derived : MethodCustomModifierCombinations
             }
         }
 
+        /// <summary>
+        /// Test copying custom modifiers in/on parameters/return types.
+        /// </summary>
+        [Fact]
+        public void TestPartialMethodOverrideCombinations()
```

Comment:
I didn't see any similar test to this already so figured we might as well have it.